src/middlewared/middlewared/client/client.py

```diff
@@ -27,6 +28,25 @@
     LIBZFS = True
 
 
+class TCPKeepAlive(enum.Enum):
+
+    KEEPIDLE = (256, 4)
+    KEEPINTVL = (512, 5)
+    KEEPCNT = (1024, 6)
```

Comment:
why not use socket.TCP_KEEP*?

Reply:
It's not exported in the socket module from what I've gathered.

Reply:
Maybe not in FreeBSD, but its there for linux at least

Reply:
yep, it is in freeBSD too....i must have typo'ed earlier. I'll fix.